mobile/src/common/enums/app/env.enum.ts

```diff
@@ -1,13 +1,14 @@
 import Config from 'react-native-config';
 
-const { API_ORIGIN_URL, APP_HOST, APP_SCHEME, STRIPE_PUBLIC_KEY } = Config;
+const { API_ORIGIN_URL, APP_HOST, APP_SCHEME, REACT_APP_STRIPE_PUBLIC_KEY } =
```

Comment:
Actually, REACT_APP prefix is needed for frontend created by cra. So, please revert to prev one

Reply:
Don`t forget to change it into .env locally